src/Symfony/Component/Security/Core/Resources/translations/security.en.xlf

```diff
@@ -74,10 +74,6 @@
                 <source>Too many failed login attempts, please try again in %minutes% minute.</source>
                 <target>Too many failed login attempts, please try again in %minutes% minute.</target>
             </trans-unit>
-            <trans-unit id="20">
-                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
-                <target>Too many failed login attempts, please try again in %minutes% minutes.</target>
-            </trans-unit>
```

Comment:
You snuck in this change too, right? 

It is removed because you could not find any use for it anymore.